### DISCUSSION TITLE
[Feature request]Toggle panel's query/view to alternative.

### DISCUSSION THREAD

**Discussion by hagen1778** [grafana/grafana#59179] [Dashboards, Panels & Visualizations]
It would be great to have the option to toggle the panel for displaying an alternative view or query.
For example, user adds a panel to show the Memory usage:
<img width="635" alt="image" src="https://user-images.githubusercontent.com/2902918/203502163-140c150f-d8b8-471c-8496-259c428efb01.png">

An alternative view for this panel would be showing the Memory usage in %:
<img width="626" alt="image" src="https://user-images.githubusercontent.com/2902918/203502382-399258cf-2c6a-43db-ba58-5a6cd5074b19.png">

Or to show the breakdown of Memory usage by instance/service/pod:

<img width="629" alt="image" src="https://user-images.githubusercontent.com/2902918/203502638-6bab7337-120e-4555-bf7a-85b155ecc310.png">

All the panels above are helpful for analysis. However, adding all three of them to the dashboard makes it too big. Would be much better to have only one panel "Memory" and a toggle button that could change/switch it to an alternative view (query, transformations, units, etc.).

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
